File: src/main/java/com/rkroom/blog/service/UserService.java
package com.rkroom.blog.service;

import com.rkroom.blog.entity.User;

public interface UserService { //UserService接口
    User selectByUsername(String username); //一个根据用户名查询用户信息的接口
    User selectById(int id);
    int updatePasswordById(String password,int id);
    void insertUser(User user);
}

File: src/main/java/com/rkroom/blog/service/UserService.kt
package com.rkroom.blog.service

import com.rkroom.blog.entity.User

interface UserService {
    //UserService接口
    fun selectByUsername(username: String?): User? //一个根据用户名查询用户信息的接口

    fun selectById(id: Int): User?
    fun updatePasswordById(password: String?, id: Int): Int
    fun insertUser(user: User?)
}